crates/core/tedge/src/cli/certificate/cli.rs

```diff
@@ -44,6 +44,10 @@ pub enum TEdgeCertCli {
 
     /// Renew the device certificate
     Renew {
+        /// CA from which the certificate will be renew
+        #[arg(value_enum, default_value = "self-signed")]
+        ca: CertRenewalCA,
+
         #[clap(subcommand)]
         cloud: Option<CloudArg>,
     },
```

Comment:
Something is wrong here.

In `tedge cert renew c8y`, `c8y` can be understood either as the CA and as the cloud profile. Even `clap` is confused:

- `tedge cert renew self-signed c8y --profile test` is correctly interpreted as a request to renew the self-signed certificate used to connect the test c8y tenant
- while `tedge cert renew c8y c8y --profile test` leads to an error, `clap` incorrectly assuming the default CA (i.e. self-signed) and being then confused by the repeated cloud parameter.